app/controllers/documents_controller.rb

```diff
@@ -61,30 +55,30 @@ def update
           "items" => @issues.items,
         }
 
-        render :edit, status: :unprocessable_entity
-        return
+        render :edit, assign: { edition: edition }, status: :unprocessable_entity
```

Comment:
This looks nice, but why the `next`?

Reply:
That something I fixed based off the linter without much thought. 

It's suggested because return leaves the whole method whereas a next leaves the block, and I took that as a sign that it was a consequence of switching to a block.

But now you've raised that I realise it always used to be in a block so I wonder if there is a special linting rule for transactions and return 😕

Reply:
Ah I see why it triggered, as per: https://www.rubydoc.info/gems/rubocop/RuboCop/Cop/Lint/NonLocalExitFromIterator

It meets these criteria: 

> - No value is returned,
> - the block is preceded by a method chain,
> - the block has arguments,

Whereas `transaction` does not meet those.